src/Components/ErrorToast/ErrorToast.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { ToastContainer, toast } from "react-toastify";
+
+import "react-toastify/dist/ReactToastify.css";
+
+export const showError: (msg: string) => React.ReactText = (msg: string) =>
+  toast.error(`❌${msg}`, {
+    position: "top-center",
+    autoClose: 5000,
+  });
+
+export const ErrorModal: React.FC = () => (
```

Comment:
As PR feedback: this file could have been its own PR, alongside the places where it is used

`showError` can get moved to the utils file though, as it's a utility function and not a Component
Also make sure your component name matches the file name - this should be `ErrorToast`